service/src/main/java/org/apache/griffin/core/job/SparkSubmitJob.java

```diff
@@ -185,17 +185,13 @@ private void setPredicates(String json) throws IOException {
         if (StringUtils.isEmpty(json)) {
             return;
         }
-        List<Map<String, Object>> maps = toEntity(json,
-                new TypeReference<List<Map>>() {
+        List<SegmentPredicate> predicates = toEntity(json,
```

Comment:
is this code covered with tests? if not, let avoid changing any code if not necessary, or provide tests

Reply:
Yes, this code is covered in tests. I add yet another.